components/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,46 @@
+import Tabs, { TabsProps } from '@mui/material/Tabs'
+
+type BreadcrumbsProps = TabsProps
```

Comment:
can do away with this its just referencing TabsProps. use TabsProps directly in ur component